docs/source/features/errors.md

```diff
@@ -114,23 +114,25 @@ application, you can use the base `ApolloError` class.
 
 ```js
 new ApolloError(message, code, additionalProperties);
-``` 
+```
 
 ## Masking and logging errors
 
-The Apollo server constructor accepts a `formatError` function that is run on each error passed back to the client. This can be used to mask errors as well as for logging.
-This example demonstrates masking (or suppressing the stacktrace):
+The Apollo server constructor accepts a `rewriteError` function that is run on each error passed back to the client or trace reporting. This can be used to mask errors as well as for logging. This example demonstrates masking (or suppressing the error):
 
-```js line=4-10
+```js line=4-12
 const server = new ApolloServer({
   typeDefs,
   resolvers,
-  formatError: error => {
+  rewriteError: error => {
```

Comment:
Just flagging that this documentation is askew/wrong right now (this is not where `rewriteError` would be defined, it would be within the `engine` property); it seems that the commits from https://github.com/apollographql/apollo-server/pull/2585 are also on this branch right now.